packages/vite/src/node/config.ts

```diff
@@ -350,6 +388,7 @@ export type ResolvedConfig = Readonly<
     worker: ResolveWorkerOptions
     appType: AppType
     experimental: ExperimentalOptions
+    serverPersistentCache: ResolvedServerPersistentCacheOptions | null
```

Comment:
We should create a `ResolvedExperimentalOptions` here, so we can have `ResolvedServerPersistentCacheOptions` in `resolvedConfig.experimental`. So we align with other experimental options.